api/ApiHandler.py

```diff
@@ -1,21 +1,34 @@
-from flask_restful import Api, Resource, request
-
-class ApiHandler(Resource):
-  def get(self):
-    search_term = request.args.get("query")
-    ## TODO: return err if query is invalid twitter handle
-    ## TODO: use search_term to return proper array of words
-    return {
-      'resultStatus': 'SUCCESS',
-      'words': [
-                {
-                    "text": 'told',
-                    "value": 64
-                },
-                {
-                    "text": "hi", 
-                    "value": 1
-                }
-            ]
-      }
+from flask_restful import Resource
+import requests;
+import os;
 
+from flask import Flask, jsonify, json
+
+
+class ApiHandelr(Resource):
```

Comment:
Should be: `class ApiHandler(Resource):`

(ApiHandler is spelled wrong)

Reply:
Done